include/enums.h

```diff
@@ -3325,4 +3326,10 @@ enum WindowGroupId {
     WINDOW_GROUP_FILE_MENU = 3,
 };
 
+enum RushFlags {
+    RUSHES_NONE       = 0,
+    RUSHES_MEGA_RUSH  = 1,
```

Comment:
Seems to be a disconnect between "RUSHES" and "Rush" here. I'd prefer we call em RUSH_ and the field `rushFlags`

Reply:
Renamed to `RUSH_FLAG_MEGA`, `RUSH_FLAG_POWER`, `RUSH_FLAG_NONE` after a discord discussion